### ISSUE TITLE
Julia not highlighted correctly

### ISSUE THREAD

**Issue by saulshanabrook** [jupyterlab/jupyterlab#4349]
Some Julia code renders differently than it does on the codemirror site. Some words are not highlighted in the Jupyterlab text editor. For example, with this code:

```julia
struct SimulationInfo
    ProximalParams::LowRankModels.ProxGradParams
    row::Int64
```

![screen shot 2018-04-10 at 7 53 55 am](https://user-images.githubusercontent.com/1186124/38555531-7af2b658-3c94-11e8-91d0-9e85a2c3e9d7.png)

You can see also that on the CodeMirror site `struct` has class `cm-keyword` but on JupyterLab it has class `cm-variable`. 

Originally raised by @jasonsun0310 on [Gitter](https://gitter.im/jupyterlab/jupyterlab?at=5acbfa626d7e07082bd8614e)

**Comment by ian-r-rose** [comment]
I'm glad you could reproduce this, as I could not on my machine. Are there any network errors related to trying to load a codemirror mode from the server?

**Comment by jasongrout** [comment]
Also, we are at codemirror version 5.35, while the latest is 5.36. Looking at the release notes, I don't imagine that explains the problem, though.

**Comment by saulshanabrook** [comment]
@ian-r-rose No, no network errors. It does work me locally with `--dev`/`--dev-mode`, but not otherwise. I have my local version installed, so I don't think it's that one of those modes is using an older code base than the other.

**Comment by ian-r-rose** [comment]
Ah, I think this was probably fixed by upgrading to CodeMirror 5.35, and we just haven't released a version with it yet. The hazards of having a longer release cycle for the beta series...

https://github.com/codemirror/CodeMirror/pull/4842

**Comment by jasongrout** [comment]
That makes more sense. Closing as fixed in master, then.
